src/getters/my_functions.py

```diff
@@ -0,0 +1,8 @@
+def add(number_one, number_two):
+    return number_one + number_two
+
+def divide(number_one, number_two):
+    if number_two == 0:
+        return None
```

Comment:
```suggestion
        raise ZeroDivisionError
```
this should be raising a ZeroDivisionError, rather than hiding an error with None

Reply:
Thanks @sqr00t!